pkg/policyGovernance/security/imageScanning/repository/ImageScanHistoryRepository.go

```diff
@@ -163,3 +164,13 @@ func (impl ImageScanHistoryRepositoryImpl) FindByImage(image string) (*ImageScan
 		Where("image = ?", image).Order("execution_time desc").Limit(1).Select()
 	return &model, err
 }
+
+func (impl ImageScanHistoryRepositoryImpl) FindByImageAndDigestWithState(imageDigest string, image string) (*ImageScanExecutionHistory, error) {
```

Comment:
change func name , impl is not acc to func name

Reply:
DONE